metrics/ci_integrations/test/cli/test_util/destination_party_stub.dart

```diff
@@ -0,0 +1,22 @@
+// Use of this source code is governed by the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+import 'package:ci_integration/integration/interface/base/client_factory/integration_client_factory.dart';
+import 'package:ci_integration/integration/interface/base/config/parser/config_parser.dart';
+import 'package:ci_integration/integration/interface/base/config/validator_factory/config_validator_factory.dart';
+import 'package:ci_integration/integration/interface/destination/client/destination_client.dart';
+import 'package:ci_integration/integration/interface/destination/config/model/destination_config.dart';
+import 'package:ci_integration/integration/interface/destination/party/destination_party.dart';
+
+/// A stub implementation of the [DestinationParty] to use in tests.
+class DestinationPartyStub extends DestinationParty {
```

Comment:
```suggestion
class DestinationPartyStub implements DestinationParty {
```
Please, apply this suggestion to all `Stub` classes.